docs/community/community-call.md

```diff
@@ -17,36 +17,44 @@ image:
 
 ## Community Call
 
-https://community.sap.com/resources/community-calls
+In this series of calls for SAP Community, we will give you regular updates on the latest releases of the SAP Cloud SDK, its roadmap, and topics around extensibility of SAP Business Suite, SAP S/4HANA, and other SAP offerings using SAP Cloud Platform. The call series will be conducted by the SAP Cloud SDK development and product management team. We will introduce SAP Cloud SDK and cover its latest updates.
+
+The community update calls take place on every first Tuesday of a month. On this page, you can find the schedule as well as a history of past sessions below.
+
+[Register for the series](https://sap-se.zoom.us/webinar/register/WN_dBtYmF5IR7Gb8TMjHqEvhw)
+
+### What do you want to learn?
+
+Our community call series is as good as the value it bring to the community. If you are interested in a particular topic, feel free to request it by contacting us and we'll do our best to deliver a session in the near future.
 
 ## Topics Backlog
 
 * Build an application using SAP Cloud SDK in SAP Business Application Studio
 
 ## Upcoming Dates 2020
```

Comment:
I think we can skip this section, only mention the next call in the section above.